### DISCUSSION TITLE
Host project in github

### DISCUSSION THREAD

**Discussion by Ramandeep97** [EddieHubCommunity/LinkFree#10248] [Q&A]
How can I host my project.
https://github.com/Ramandeep97/Freelancers

**Comment by Vijaykv5** [comment]
Hi @Ramandeep97 
You can host your project on platforms like GitHub Pages, Heroku, or Netlify. Simply push your code to a repository on GitHub, then set up deployment. 
Netlify, vercel and render are best service platforms where you can host your personal projects for free.

**Reply by Ramandeep97** [reply]
thnx
